package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "ts-node": "TS_NODE_TRANSPILE_ONLY=true ts-node --project storybook/tsconfig.storybook.json",
     "storybook": "yarn install && yarn ts-node node_modules/.bin/start-storybook -c storybook",
-    "storybook:build": "yarn ts-node node_modules/.bin/build-storybook -c storybook -o storybook/public",
+    "storybook:build": "NODE_ENV=production yarn ts-node node_modules/.bin/build-storybook -c storybook -o storybook/public",
```

Comment:
Last question, mostly out of curiosity: is this fixing something? ☝️

Reply:
Just making sure I don't add the storybook header and footer for local builds: https://github.com/cultureamp/kaizen-design-system/pull/656/files#diff-85d77a5014e178e22ecd89cd8866a047R8